vendor/css/carousel.css

```diff
@@ -0,0 +1,78 @@
+.jcarousel-skin .jcarousel-container {
+	background: #FFF;
```

Comment:
file này cũng sai indent 2 space cho toàn file luôn, review và fix cho các file tương tự